### DISCUSSION TITLE
[Motion] Only animate the parent element and not its content

### DISCUSSION THREAD

**Discussion by LasseRosenow** [lit/lit#2333] [Q&A]
I recently tried the motion package from lit-labs and wanted to animate a box, that changed its size.

The issue is, that there is text in the box and when the box animates its height for example, the text gets stretched.

Is there any way to only animate the height of the parent while not animating the children within?

**Comment by kevinpschaaf** [comment]
Currently height/width animations use `scale`. We could consider adding an option for animating height/width, but then we'd need to consider how to avoid text reflow (which is both expensive and can also be a bad user experience with unexpected wrapping).
